Fix more than one module parameters parsing (char & string types)

--- a/modutils/modutils-24.c	2010-07-25 22:51:49.000000000 +0400
+++ b/modutils/modutils-24.c	2011-02-10 21:34:23.000000000 +0300
@@ -2522,9 +2522,11 @@ new_process_module_arguments(struct obj_
 						     *pinfo, param);
 			}
 
-			p = skip_whitespace(p);
-			if (*p != ',')
-				break;
+			if (*p != '\0') {
+				p = skip_whitespace(p);
+				if (*p != ',')
+					break;
+			}
 			p = skip_whitespace(p + 1);
 		}
 
-- 
